packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx

```diff
@@ -220,7 +241,12 @@ export default createFragmentContainer(AgendaItem, {
       endedAt
       facilitatorStageId
       facilitatorUserId
+      phases {
+        phaseType
+        ...AgendaItemAllMeetingPhases @relay(mask: false)
```

Comment:
We could pass in `...AgendaItemPhase` instead here and then we'd be able to remove `...AgendaItemAllMeetingPhases` from the file